style.css

```diff
@@ -0,0 +1,139 @@
+#top-nav {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+}
+
+#top-nav a {
+  text-decoration: none;
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 24px;
+}
+
+#top-nav a:hover {
+  text-decoration: underline;
+  text-decoration-color: #23a3fa;
+  text-decoration-thickness: 2px;
+}
+
+#top-nav nav {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+#home_content {
+  background-image:
+    url("img/Ellipse 11.png"),
+    url("img/Rectangle 6.png"),
+    url("img/Rectangle\ 4.png"),
+    url("img/Ellipse 3.png"),
+    url("img/Ellipse\ 5.png"),
+    url("img/Rectangle\ 7.png"),
+    url("img/Ellipse 12.png"),
+    url("img/Ellipse 4.png"),
+    url("img/Rectangle\ 4.png"),
+    url("img/Ellipse\ 13.png"),
+    url("img/Ellipse 5.png");
+  background-position:
+    19px 90px,
+    9px 15px,
+    214px 110px,
+    323px 102px,
+    125px 106px,
+    127px 98px,
+    370px 385px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
+}
+
+#home_content h1 {
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: start;
+  margin: 0 24px;
+}
+
+#home_content p {
+  font-family: Poppins, sans-serif;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 24px 0 24px;
+}
+
+#home_content p:hover {
+  text-decoration: underline;
+  text-decoration-color: #23a3fa;
+  text-decoration-thickness: 2px;
+}
```

Comment:
- [OPTIONAL] Kindly make sure you use the hover effect on clickable elements only to make your project professional :recycle: